### DISCUSSION TITLE
Manage connection of line (ERD)

### DISCUSSION THREAD

**Discussion by hairullana** [dbeaver/dbeaver#23216] [General]
In an Entity-Relationship Diagram (ERD), how can I visually represent the relationships between tables using lines that connect the foreign key field in one table to the primary key field in the related table?

In the latest version/update, the connection line is now correct and automatically links to the appropriate fields (foreign key and primary key) without requiring the user to click on the line first.

![image](https://github.com/dbeaver/dbeaver/assets/56705867/55352b04-e9de-40c9-b3cb-7be9cca25235)

**Comment by LonwoLonwo** [comment]
Hello @hairullana 

Sorry, it is unclear for now - do you want something else? An extra line? Or what? Could you please draw it?

**Reply by hairullana** [reply]
Hello @LonwoLonwo 

I mean like this, the line is connecting between parent key and foreign key
![image](https://github.com/dbeaver/dbeaver/assets/56705867/b8748e47-ebc6-452e-aefc-c7db6e83e731)
